Automattic/themes-workspace
# Add additional Botany default styles to Varya

**Description**

This PR adds additional styles for the [Botany theme](https://www.figma.com/file/eBaFrMSiiJxEeUU6T5LU2I/Botany?node-id=71%3A49) to the work started by @allancole. 

<img width="1032" alt="Screen Shot 2020-03-25 at 6 18 20 PM" src="https://user-images.githubusercontent.com/5375500/77591760-0898cb80-6ec7-11ea-80b3-0c33cedfc8a9.png">

**To Test**
- Check out this PR
- Upload and activate this theme
- Create some test posts and pages. You can borrow from some layouts and content from the pressable staging site: https://botanytheme.mystagingwebsite.com/

Next steps would be to review and determine which changes to integrate directly into the theme, rather than having them enqueued as "default styles" within the theme.
